bundler.d/assets.rb

```diff
@@ -1,21 +1,18 @@
 group :assets do
   gem 'ace-rails-ap', '~> 4.0.0'
-  gem 'sass-rails', '~> 3.2'
+  gem 'sass-rails', '~> 4.0'
   gem 'uglifier', '>= 1.0.3'
   gem 'execjs', '>= 1.4.0', '<2.5.0'
-  gem 'jquery-rails', '2.0.3'
+  gem 'jquery-rails', '~> 2.0.3'
   gem 'jquery-ui-rails', '< 5.0.0'
   gem 'bootstrap-sass', '3.0.3.0'
-  gem 'spice-html5-rails', '~> 0.1.4'
+  gem 'spice-html5-rails', '~> 0.1.5'
   gem 'flot-rails', '0.0.3'
   gem 'quiet_assets', '~> 1.0'
-  gem 'gettext_i18n_rails_js', '~> 0.0', '>= 0.0.8'
-  # unspecified dep of gettext_i18n_rails_js
-  #   https://github.com/nubis/gettext_i18n_rails_js/pull/23
-  gem 'gettext', '~> 3.1', :require => false
+  gem 'gettext_i18n_rails_js', '~> 1.0'
   gem 'multi-select-rails', '~> 0.9'
   gem 'gridster-rails', '~> 0.5'
-  gem 'jquery_pwstrength_bootstrap', '~> 1.2'
+  gem 'jquery_pwstrength_bootstrap', github: "unorthodoxgeek/jquery_pwstrength_bootstrap-gem" #TODO
```

Comment:
I don't have permissions to push this gem. https://github.com/dragonfly9517/jquery_pwstrength_bootstrap-gem/pull/1 needs to be merged and then pushed to rubygems.org